status: improve descriptions of host stats

RaduBerinde opened the pull request:
This commit adds a `(as reported by the OS)` to the description of metrics that are obtained from the operating system. We have encountered many instances where someone was asking for clarification on this.

Epic: none
Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/115501)
<!-- Reviewable:end -->

RaduBerinde committed:
status: improve descriptions of host stats

This commit adds a `(as reported by the OS)` to the description of
metrics that are obtained from the operating system. We have
encountered many instances where someone was asking for clarification
on this.

Epic: none
Release note: None

aadityasondhi reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/115501#-:-Nkq3dqM7lfkVZ5q2YNQ:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @abarganier and @jbowens)



<!-- Sent from Reviewable.io -->

RaduBerinde commented:
TFTR!

bors r+
